src/coreclr/jit/simdcodegenxarch.cpp

```diff
@@ -1987,9 +1966,8 @@ void CodeGen::genSIMDIntrinsicSetItem(GenTreeSIMD* simdNode)
         assert(genIsValidIntReg(tmpReg));
 
         // Move the value from xmm reg to an int reg
-        instruction ins = ins_CopyFloatToInt(TYP_FLOAT, TYP_INT);
-        // (Note that for mov_xmm2i, the int register is always in the reg2 position.
-        inst_RV_RV(ins, op2Reg, tmpReg, baseType);
+        // Another error, op2Reg is our xmm source but it was passed second.
```

Comment:
I guess you want to delete this comment before merging?

Reply:
yeah, I thought I did, thanks.